_posts/2020-11-09-pulling-most-recent-files.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Pull most recent files in R"
+date:   2020-11-09
+categories: jekyll update
```

Comment:
"Categories" should include tags related to your blog post. You could change this from “jekyll update” to “R” and any other related tags.

Reply:
changed to R base packages